src/typescript-generator/script.js

```diff
@@ -146,7 +161,15 @@ export class Script {
   exportStatements() {
     return Array.from(
       this.exports.values(),
-      ({ name, isType, code, typeDeclaration }) => {
+      ({ name, isType, isDefault, code, typeDeclaration }) => {
+        if (code.raw) {
+          return code.raw;
+        }
```

Comment:
This line is for a special case where the export line looks like this:

```ts
export { default } from "../$context.js";
```

I was hoping I could refactor before merging this PR, but I'm not seeing a clear path, so I'll leave it for now. Need to go back and reacquaint myself with the relationship between `Coder` and `Script`.